postgresql/README.md

```diff
@@ -1,10 +1,80 @@
 # PostgreSQL
 
+This packages PostgreSQL in a Habitat package that can be run with the Habitat supervisor.
+
 This PostgreSQL plan supports standalone and clustered modes as well as continuous archiving via wal-e integration.
 
-*NOTE:* The current version of this plan is an intterim version of the package that cannot be considered production ready. It can be used however archiving (by default) copies data in `{{pkg.svc_path}}/archive` which cannot be guaranteed as persisted between failures. That being said, the archive location is tunable in the default.toml.
+*NOTE:* The current version of this plan is an interim version of the package that cannot be considered production ready. It can be used however archiving (by default) copies data in `{{pkg.svc_path}}/archive` which cannot be guaranteed as persisted between failures. That being said, the archive location is tunable in the default.toml.
```

Comment:
Is this still true? Don't we use this in production?

Reply:
Good point, just confirmed that is no longer true, so removing this section!